packages/types/src/user.ts

```diff
@@ -1,7 +1,7 @@
 /** JSDoc */
 export interface User {
   [key: string]: any;
-  id?: string;
+  id?: string | number;
```

Comment:
Looks like relay theoretically allows numbers here:

https://github.com/getsentry/relay/blob/4a7c73a1e3e00215929b4b76644043771517c08c/relay-general/src/protocol/user.rs#L49

`LenientString` is like python's `str()`. We should still double check this.

Reply:
LenientString allows _any_ json type